src/styles/styles.js

```diff
@@ -2751,6 +2751,18 @@ const styles = {
         alignSelf: 'center',
     },
 
+    dragAndDropOverlay: {
```

Comment:
I think we could rename this style to make it a bit more generic maybe? 

```suggestion
    fullScreenTransparentOverlay: {
```